src/uikit/Select.js

```diff
@@ -3,91 +3,179 @@ import Downshift from 'downshift';
 
 import downChevronIcon from '../assets/icon-chevron-down-grey.svg';
 
+export const OptionDropdownWrapperCss = `
```

Comment:
seems like a bit of an odd pattern for CSS, but maybe that'll be part of the refactor? regardless should probably be a lowercase var name as it's not a component